ocs_sample_library_preview/Security/UserInvitation.py

```diff
@@ -0,0 +1,177 @@
+import json
+
+from .UserInvitationState import UserInvitationState
+
+
+class UserInvitation(object):
+    """OCS trustee definition"""
+
+    def __init__(self, id: str = None, expires_date_time: str = None, issued: str = None, expires: str = None, accepted: str = None,
+                 state: UserInvitationState = None, send_invitation: bool = None, identity_provider_id: str = None, tenant_id: str = None, user_id: str = None):
+        self.Id = id
+        self.ExpiresDateTime = expires_date_time
+        self.Issued = issued
+        self.Expires = expires
+        self.Accepted = accepted
+        self.State = state
+        self.SendInvitation = send_invitation
+        self.IdentityProviderId = identity_provider_id
+        self.TenantId = tenant_id
+        self.UserId = user_id
+    
+    @property
+    def Id(self) -> str:
+        return self.__id
+
+    @Id.setter
+    def Id(self, value: str):
+        self.__id = value
+
+    @property
+    def ExpiresDataTime(self) -> str:
```

Comment:
should this be `ExpiresDat**e**Time`?

Reply:
huh, can't embed **bolded characters** inside of `blocks`. fun fact. I meant to say 'ExpiresDat**e**Time'

Reply:
Yes, it should. Good catch!